README.md

```diff
@@ -145,9 +151,9 @@ In each of the below commands, you should pass `atomizer-compose.cfg` instead of
 
 Running Unit & Integration Tests
 
-1. Build the container
+1. Build the container. Target `builder` stage as the tests need the dependencies.
```

Comment:
```suggestion
1. Build the container.
   Target `builder` stage as the tests need the dependencies.
```

This is a bit of a nitpick, but one-sentence-per-line makes storing documentation in-VCS dramatically nicer in the long run.